Add-link-to-tracing-workshop (#11009)
getsentry/sentry-docs: docs/concepts/key-terms/tracing/distributed-tracing.mdx

A tracing tool, on the other hand, focuses on _what_ happened (and when), rather than how many times it happened or how long it took. The resulting trace is a log of events which occurred during a program's execution, often across multiple systems. Though traces most often - or, in the case of Sentry's traces, always - include timestamps, allowing durations to be calculated, measuring performance is not their only purpose. They can also show the ways in which interconnected systems interact, and the ways in which problems in one can cause problems in another.

> To learn more, check out this on-demand [Tracing: Frontend issues with backend solutions](https://sentry.io/resources/tracing-frontend-issues-with-backend-solutions/?original_referrer=https%3A%2F%2Fblog.sentry.io%2F) workshop, put on by the folks at Sentry.

Reviewer: ```suggestion
To learn more, watch this [Tracing: Frontend issues with backend solutions](https://sentry.io/resources/tracing-frontend-issues-with-backend-solutions/?original_referrer=https%3A%2F%2Fblog.sentry.io%2F) workshop.
```
I'd avoid "check out" for non-native speakers and I don't think it adds much to say that it's put on by folks at Sentry.

Author: Great points again :)
